src/scripts/components/foundations/Text.js

```diff
@@ -47,8 +49,16 @@ export default styled.p`
       return props.theme.fonts.text.mainLineHeight
     }
   }};
+  opacity: ${props => (props.disabled ? 0.5 : null)};
   overflow-wrap: break-word;
+  text-transform: ${({ textTransform }) => textTransform || null};
   white-space: pre-wrap;
   word-wrap: break-word;
   margin: ${({ margin }) => margin};
 `
+
+export const Strong = styled.strong`
+  ${ColorText} font-weight: ${props => props.theme.fonts.weight.bold};
```

Comment:
Can you add a newline between these styles? Without them it is difficult to tell what is going on

Reply:
@bent0b0x, this changes on `git commit...`